src/Microsoft.ML.DnnImageFeaturizer.ResNet18/ResNet18Extension.cs

```diff
@@ -22,7 +22,14 @@ public static class ResNet18Extension
         /// </summary>
         public static EstimatorChain<ColumnCopyingTransformer> ResNet18(this DnnImageModelSelector dnnModelContext, IHostEnvironment env, string outputColumnName, string inputColumnName)
         {
-            return ResNet18(dnnModelContext, env, outputColumnName, inputColumnName, Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "DnnImageModels"));
+#if NETFRAMEWORK
```

Comment:
I don't understand how this works/fixes anything. We aren't multi-targeting this assembly.

Reply:
We should be able to do this using only `netstandard2.0` APIs. We shouldn't need to multi-target to make this work.